include/functions.h

```diff
@@ -915,7 +915,7 @@ void func_800BDC5C(SkelAnime* skelAnime, ActorAnimationEntry animation[], s32 in
 void Actor_Noop(Actor* actor, GlobalContext* globalCtx);
 void func_800BDFC0(GlobalContext* globalCtx, Gfx* dl);
 void func_800BE03C(GlobalContext* globalCtx, Gfx* dl);
-// void func_800BE0B8(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
+Actor *func_800BE0B8(GlobalContext *globalCtx, Actor *inActor, s16 arg2, u8 arg3, f32 arg4);
```

Comment:
```suggestion
Actor* func_800BE0B8(GlobalContext* globalCtx, Actor* inActor, s16 arg2, u8 arg3, f32 arg4);
```
Just as a follow-up. Even if the formatter doesn't run on header files, it's still good to make sure it's in good format

Reply:
Agreed. Can you please update this?